src/link/constants.scss

```diff
@@ -118,7 +118,7 @@ $link-styles: (
       'text-color-default': awsui.$color-text-link-button-normal-default,
       'text-color-hover': awsui.$color-text-link-button-normal-hover,
       'text-color-active': awsui.$color-text-link-button-normal-active,
-      'font-weight': awsui.$font-button-weight,
+      'font-weight': awsui.$font-weight-button,
```

Comment:
Uncertain here. I could see arguments both ways, but I think this is still semantically more a link than a button, and like for `info` the weight is an important affordance as there is no border/background

Reply:
Resolved!